packages/core-connector-interface/src/indexer.d.ts

```diff
@@ -1,12 +1,25 @@
-import { Balance } from './types'
+import type { Balance, Public, ChannelEntry } from './types'
 
-type Channel = [source: PeerId, destination: PeerId, stake: Balance]
+export type RoutingChannel = [source: PeerId, destination: PeerId, stake: Balance]
```

Comment:
I don't love the name 'RoutingChannel' tbh.

We are totally overloading what a Channel is.

I would love a single Channel interface that captures it's directed nature.

Reply:
agree! lets fix that in the refactor
I renamed it from `IndexerChannel` to `RoutingChannel` since that type is only used in routing where the rest of the code uses `ChannelEntry`

Reply:
👍